apps/115-remove-modal/tests/shinytest/mytest.R

```diff
@@ -0,0 +1,11 @@
+app <- ShinyDriver$new("../../", seed = 100, shinyOptions = list(display.mode = "normal"))
+app$snapshotInit("mytest")
+
+app$setInputs(openModalBtn = "click")
+Sys.sleep(1)
+app$snapshot()
+
+# Verify modal is closed
+app$setInputs(closeModalBtn = "click")
+Sys.sleep(1)
+app$snapshot()
```

Comment:
```suggestion
app$setInputs(openModalBtn = "click")
app$executeScript(
  "window.modalShown = false;
  $(document).on('shown.bs.modal', function(e) { window.modalShown = true; });"
)
app$snapshot()

# Verify modal is closed
app$setInputs(closeModalBtn = "click")
app$executeScript(
  "window.modalShown = false;
  $(document).on('shown.bs.modal', function(e) { window.modalShown = true; });"
)
app$snapshot()
```

Reply:
Modified the wait for modal dialog to be opened or closed based on discussion with Carson. Referred to https://github.com/rstudio/shinycoreci-apps/blob/4a2e7854e588e04846a8aca08ecd635616634095/apps/301-bs-themes/tests/shinytest/custom3.R#L56-L62 and https://getbootstrap.com/docs/4.0/components/modal/